File: corpus/2018q1/_p2q1_ok/_check/A;0126.py

int n = 125
int k = 30
bool resultado = false
for a=0 in range(0, n):
    Scanner entrada = new Scanner (System.in)
    print("Digite um número inteiro: ")
resultado = comparaVetor(v, n, k)
if resultado=true:
    print("Existe número menor a 30 nessa vetor. ")
    }else{
    print("Não existe número menor a 30 nessa vetor. ")
def comparaVetor(int v[], int n, int k):
    bool result = false
    for a=0 in range(0, n):
        if v[a]< k:
            result = true
    return result

File: corpus/2018q1/_p2q1_ok/_check/A;0126.java

import java.util.Scanner;

public class Exercicio1 {
  public static void main(String[] args) {

  int n = 125; 
  int k = 30;
  int v[] = new int[n];
  boolean resultado = false;

for(int a=0; a<n; a++){
Scanner entrada = new Scanner (System.in); 
System.out.println("Digite um número inteiro: ");
v[a]=entrada.nextInt();
}
resultado = comparaVetor(v, n, k);     
if(resultado=true){
  System.out.println("Existe número menor a 30 nessa vetor. ");

  }else{
      System.out.println("Não existe número menor a 30 nessa vetor. ");

 }
}
    public static boolean comparaVetor(int v[], int n, int k){
    boolean result = false; 

    for(int a=0; a<n; a++){
    if(v[a]< k){
     result = true;
        }
     }
    return result;
  }

}